.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,55 @@
+# Implementing a new supported platform
+
+## At Development
+Create PR's addressing the following:
+- [ ] [coreos-assembler](https://github.com/coreos/coreos-assembler)
+    - [Example PR](https://github.com/coreos/coreos-assembler/pull/2489)
+    - [ ] `Implement required functionality to support new platform`
+- [ ] [fedora-web](https://pagure.io/fedora-web/websites)
+    - [Example PR](https://pagure.io/fedora-web/websites/pull-request/221#request_diff)
+    - [ ] `Add platform to the sites/static/js/coreos-download.js`
+- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
```

Comment:
Also need to update the metadata docs in fedora-coreos-tracker.

Reply:
Its been a month, but  I don't remember creating a pr for fedora-coreos-tracker. 

I will take another look shortly.

Reply:
Yeah, it's a manual update of the three files changed in that commit.